Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -247,11 +259,13 @@ - (void)stopAnimating {
     RunSynchronouslyOnMainThread(^{
         if (_isAnimating) {
             _isAnimating = NO;
-        }
 
-        _progressRing.IsActive(false);
-        if (_hidesWhenStopped) {
-            [self setHidden:YES];
+            if (_hidesWhenStopped) {
+                // Make our view transparent, rather than hidden, to ensure space is reserved in the layout.
+                [self setAlpha:0.0];
```

Comment:
Ah, yeah. We could handle for that, but since this is also another edge case (user explicitly setting alpha on the entire view, not just the indicator itself) it may not be worth it. Since the ProgressRing already does not display anything when it is not active, this is technically handling for another edge case: the user explicitly setting a background on the indicator view. If a background is set, it would not be hidden when the animation is stopped since we don't hide the view in order to preserve space in the layout.
Now the question is, do we try to handle for either case, or simply leave it as a caveat that if the user wants to use a background in the indicator then they are responsible for hiding it when animation stops as well?

---
In reply to: [113080688](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113080688) [](ancestors = 113080688)